managers/views.py

```diff
@@ -92,9 +92,12 @@ class ProjectCreateView(CreateView):
     model = Project
     template_name = "managers/project_form.html"
 
+    def get_form(self, form_class: ProjectAdminForm = None) -> ProjectAdminForm:
+        ProjectAdminForm.user_pk = self.request.user.pk
```

Comment:
This should be passed in `get_form_kwargs` method and handled in form `__init__`.

Reply:
Not sure about my solution but I think i resolve it